examples/dedicated_vmseries/README.md

```diff
@@ -187,8 +187,8 @@ terraform destroy
 ### Providers
 
 - `random`
-- `azurerm`
 - `local`
+- `azurerm`
```

Comment:
Can you discard this change in the example README? This README shouldn't be affected by your changes and I suppose it got modified due to some problem with `terraform_docs`.